### PR TITLE
added backend

### PR DESCRIPTION
**Pull request by Yash-Var**

### CODE REVIEW

File: Rakathon/views.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 from plants.models import Plants
 from blog.models import Blogs
 from django.shortcuts import get_object_or_404
-
+# from contact_queries.models import User_Messages
```
Comment (RyanWalker277) [inline Rakathon/views.py]: remove this comment

File: Templates/contact.html [hunk 2]
```diff
@@ -557,26 +558,27 @@ <h2>GET IN TOUCH</h2>
...
                     <!-- Contact Form Area -->
                     <div class="contact-form-area mb-100">
-                        <form action="#" method="post">
+                        <form action="{% url 'submit_message' %}" method="post">
+                            {% csrf_token %}
                             <div class="row">
                                 <div class="col-12 col-md-6">
                                     <div class="form-group">
-                                        <input type="text" class="form-control" id="contact-name" placeholder="Your Name">
+                                        <input type="text" class="form-control" name="yname" id="contact-name" placeholder="Your Name">
```
Comment (RyanWalker277) [inline Templates/contact.html:566]: change name="yname" to name = "name"

File: contact_queries/urls.py [hunk 3]
```diff
@@ -3,6 +3,6 @@
 from .views import *
 
 urlpatterns = [
-    path('submit', views.contact , name = "submit_message"),
-    path('', views.contact_index , name = "submit_message"),
+    path('submit', views.contact_index , name = "submit_message"),
+    # path('', views.contact_index , name = "submit_message"),
```
Comment (RyanWalker277) [inline contact_queries/urls.py]: delete this comment

File: contact_queries/views.py [hunk 4]
```diff
@@ -1,17 +1,26 @@
...
 from django.http import HttpResponseRedirect
 
-def contact(request):
-        name = request.POST.get('name')
-        email = request.POST.get('email')
-        subject = request.POST.get('subject')
-        description = request.POST.get('description')
-        en = User_Messages(name = name , email = email , Subject = subject , Description = description )
-        en.save()
+# def contact(request):
```
Comment (RyanWalker277) [inline contact_queries/views.py]: remove these comments

**Review by RyanWalker277 — CHANGES_REQUESTED** [review]
Kindly do the necessary changes

File: contact_queries/views.py [hunk 5]
```diff
@@ -1,17 +1,26 @@
...
+#         name = request.POST.get('name')
+#         email = request.POST.get('email')
+#         subject = request.POST.get('subject')
+#         description = request.POST.get('description')
+#         en = User_Messages(name = name , email = email , Subject = subject , Description = description )
+#         en.save()
 
 def contact_index(request):
+        if request.method=="POST":
+                name = request.POST.get('contact-name')
```
Comment (RyanWalker277) [inline contact_queries/views.py:11]: change "contact-name" to "name"

File: contact_queries/views.py [hunk 6]
```diff
@@ -1,17 +1,26 @@
...
+                email = request.POST.get('contact-email')
```
Comment (RyanWalker277) [inline contact_queries/views.py:12]: change "contact-email" to "email"

File: contact_queries/views.py [hunk 7]
```diff
@@ -1,17 +1,26 @@
...
+                subject = request.POST.get('contact-subject')
```
Comment (RyanWalker277) [inline contact_queries/views.py:13]: change "contact-subject" to "subject"

**Comment by Yash-Var** [comment]
@RyanWalker277 i am closed pr make a new PR

**Pull request closed without merging**